junit-jupiter-api/src/main/java/org/junit/jupiter/api/DisplayNameGenerator.java

```diff
@@ -105,13 +105,38 @@ public String generateDisplayNameForMethod(Class<?> testClass, Method testMethod
 	}
 
 	/**
-	 * {@code DisplayNameGenerator} that replaces underscores with spaces.
+	 * Simple {@code DisplayNameGenerator} that removes trailing parentheses
+	 * for methods with no parameters.
 	 *
 	 * <p>This generator extends the functionality of {@link Standard} by
+	 * removing parentheses ({@code '()'}) found at the end of method names
+	 * with no parameters.
+	 */
+	class Simple extends Standard {
+
+		@Override
+		public String generateDisplayNameForMethod(Class<?> testClass, Method testMethod) {
+			String displayName = testMethod.getName();
+			if (hasParameters(testMethod)) {
+				displayName += ' ' + parameterTypesAsString(testMethod);
```

Comment:
❓ Why the extra space?

Reply:
I was actually wondering whether to put it or not.
I originally added it in `ReplaceUnderscores` to make `some_test(Object o)` become `some test (Object)` which looks better imo.
But here if `Simple` is used in Kotlin (I know it’s not dedicated to Kotlin usage but it was the purpose of the original request and I guess most people will use it for that), it will display such method as `some test(Object)` if there is no extra space, which I find less pretty (I might be nitpicking here).
What do you think?

Reply:
I think that's a fair point, I hadn't thought of that. 👍